django_ses_plus/models.py

```diff
@@ -19,7 +19,7 @@ class SentEmail(models.Model):
     to_email = models.EmailField()
     from_email = models.EmailField()
     subject = models.TextField()
-    html = models.FileField(upload_to=sent_email_upload_path)
+    html = models.FileField(upload_to=sent_email_upload_path, storage=get_file_storage_backend)
```

Comment:
Does this field accept callable? I though it's expecting instance of a storage or none.

Reply:
It accepts callable, I checked the source code and also tested in one of our projects.

Reply:
Great, thank you.

Reply:
It calls the callable and gets an instance of subclass of django.core.files.storage.Storage.